src/org/zaproxy/zap/extension/fuzz/resources/Messages.properties

```diff
@@ -438,6 +438,7 @@ fuzz.httpfuzzer.processor.scriptProcessor.error.persistMessageForAlert = An erro
 fuzz.httpfuzzer.processor.scriptProcessor.panel.script.label = Script:
 fuzz.httpfuzzer.processor.scriptProcessor.panel.warnNoScript.message = No script selected, a script must be selected first.
 fuzz.httpfuzzer.processor.scriptProcessor.panel.warnNoScript.title = No Script Selected
+fuzz.httpfuzzer.processor.scriptProcessor.panel.warn.title = Incorrect input
```

Comment:
Should be Title Caps: https://github.com/zaproxy/zaproxy/wiki/DevGuidelines#text-titles-menu-items-options-and-tool-tips